en/release-notes.md

```diff
@@ -25,6 +25,96 @@ The official Python client library for MAAS is available in the Ubuntu 18.04
 LTS package archive or you can download the source from:
 [https://github.com/maas/python-libmaas/releases](https://github.com/maas/python-libmaas/releases)
 
+## 2.4.0 (alpha2)
+
+### NTP services now provided by Chrony
+
+Starting with 2.4 Alpha 2, and in common with changes being made to Ubuntu Server, MAAS replaces ‘ntpd’ with Chrony for the NTP protocol. MAAS will handle the upgrade process and automatically resume NTP service operation.
+
+### Vanilla CSS Framework Transition
+
+MAAS 2.4 is undergoing a Vanilla CSS framework transition to a new version of vanilla, which will bring a fresher look to the MAAS UI. This framework transition is currently work in progress and not all of the UI have been fully updated. Please expect to see some inconsistencies in this new release.
```

Comment:
"new version of Vanilla,"
"a work in progress"

Reply:
:ok: